app/Resources/translations/messages.de.xliff

```diff
@@ -199,6 +199,18 @@
                 <source>post.no_posts_found</source>
                 <target>Keine Beiträge gefunden.</target>
             </trans-unit>
+            <trans-unit id="post.created_successfully">
+                <source>post.created_successfully</source>
+                <target>Beitrag erstellt!</target>
+            </trans-unit>
+            <trans-unit id="post.updated_successfully">
+                <source>post.updated_successfully</source>
+                <target>Beitrag bearbeitet!</target>
+            </trans-unit>
+            <trans-unit id="post.deleted_successfully">
+                <source>post.deleted_successfully</source>
+                <target>Beitrag gelöscht!</target>
```

Comment:
We could do that (both forms are correct). Though in the English version we don't use complete sentences too.

@OskarStark What do you think?